src/BikramSambat.ts

```diff
@@ -87,4 +90,18 @@ export default class BikramSambat {
   public getNextYear(): number {
     return this.year ? this.year + 1 : NaN
   }
+
+  public getWeekdayNames(language: string): string[] {
+    const languageKey = language === 'np' ? 'np' : 'en'
+    return NepaliDaysData.map((day) => day[languageKey])
+  }
+
+  public getMonthNamesNepali(language: string): string[] {
+    const languageKey = language === 'np' ? 'np' : 'en'
+    return NepaliMonthsData.map((month) => month[languageKey])
+  }
+
+  public getMonthNamesEnglish(): string[] {
+    return NepaliMonthsData.map((month) => month.en)
+  }
```

Comment:
Rather than separating into two methods, you can achieve the same behaviour considering `np` as the default like below:

```suggestion
  public getMonthNames(language?: string): string[] {
    return NepaliMonthsData.map((month) => month[language ?? 'np'])
  }
```

Reply:
That English denotes jun/july kind of english here